packages/core/src/modules/vc/jwt-vc/W3cJwtCredentialService.ts

```diff
@@ -308,13 +309,21 @@ export class W3cJwtCredentialService {
       const proverPublicKey = getKeyFromVerificationMethod(proverVerificationMethod)
       const proverPublicJwk = getJwkFromKey(proverPublicKey)
 
+      const getTrustedCertificatesForProof = agentContext.dependencyManager.isRegistered(X509ModuleConfig)
+        ? agentContext.dependencyManager.resolve(X509ModuleConfig).getTrustedCertificatesForProof
```

Comment:
If we add the `verificationContextt`,  I think `getTrustedCertificatesForVerification`. makes sense here?